src/features/Configuration/boardDefinitions.ts

```diff
@@ -36,13 +37,17 @@ const typednrf54h20json = nrf54h20pdk080json as BoardControllerConfigDefinition;
 const typednrf54h20v070json =
     nrf54h20v070json as BoardControllerConfigDefinition;
 const typednrf9151v020json = nrf9151v020json as BoardControllerConfigDefinition;
+const typednrf54l20v010json =
+    nrf54l20v010json as BoardControllerConfigDefinition;
 
 export function getBoardDefinition(
     device: Device,
     boardRevision: string | undefined
 ): BoardDefinition {
     switch (device?.devkit?.boardVersion) {
         case 'PCA10156':
+            // TODO: remove this placeholder; does not belong here
```

Comment:
Still Todo?

Reply:
yes, after tests are done, as tested 54L20 with 54L15 in this case